app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+    print(parts)
+    source_path = parts[1]
+    destination_path = parts[2]
+
+    if source_path == destination_path:
+        return
+
+    with open(source_path, "r") as file_in, \
+            open(destination_path, "w") as file_out:
+        for line in file_in:
+            file_out.write(line)
```

Comment:
You do not need to make a `for` loop here, use the `.writelines` method instead.